sink-connector/src/main/java/com/altinity/clickhouse/sink/connector/ClickHouseSinkConnectorConfig.java

```diff
@@ -263,6 +263,17 @@ static ConfigDef newConfigDef() {
                     1,
                     ConfigDef.Width.NONE,
                     ClickHouseSinkConnectorConfigVariables.REPLACING_MERGE_TREE_DELETE_COLUMN.toString())
+                .define(
+                    ClickHouseSinkConnectorConfigVariables.SOURCE_DATETIME_TIMEZONE.toString(),
+                    Type.STRING,
+                    "UTC",
```

Comment:
not sure it is a good idea. leave it `null` by default and initialize with source timezone from Debezium.
Add to the default config.yaml (commented)